.github/workflows/ci.yml

```
@@ -18,13 +18,29 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - "3.8"
           - "3.9"
           - "3.10"
+          - "3.11"
+          - "3.12"
+          - "3.13"
         django-version:
-          - django~=3.2.0
-          - django~=4.1.0
           - django~=4.2.0
+          - django~=5.0.0
+          - django~=5.1.0
+          - django~=5.2rc0
+        exclude:
+          # Django 5.0+ requires Python >=3.10
+          - python-version: "3.9"
+            django-version: django~=5.0.0
+          - python-version: "3.9"
+            django-version: django~=5.1.0
+          - python-version: "3.9"
+            django-version: django~=5.2rc0
+          # Python 3.13 supported only in Django >=5.1.3
+          - python-version: "3.13"
+            django-version: django~=4.2.0
```

If python 3.13 supports only Django>=5.1.3, then why are we using Django 4.2 with it?
```suggestion
            django-version: django~=5.1.3
```